src/store/metadata/stub/content/editions/additions-to-the-housing-stock_2023-05.json

```diff
@@ -0,0 +1,162 @@
+      "publisher": "https://staging.idpd.uk/publishers/department-for-levelling-up-housing-and-communities",
+      "creator": "office-for-national-statistics",
+      "contact_point": {
+          "name": "Department for Levelling Up, Housing and Communities",
+          "email": "mailto:correspondence@levellingup.gov.uk"
+        },
+        "topics": [
+          "https://staging.idpd.uk/topics/housing-planning-and-local-services"
+        ],
+      "frequency": "annual",
+      "keywords": [
+          "new-building",
+          "conversion",
+          "demolition",
+          "dwelling",
+          "subnational",
+          "local-authority",
+          "county"
+  
+        ],
+      "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+      "issued": "2023-05-05T08:59:11Z",
+      "modified": "2023-12-06T15:10:00Z",
+      "spatial_resolution": [
+        "E0", "E1", "E9"  
+        ],
+        "spatial_coverage": "E92000001",
+        "temporal_resolution": [
+          "P1Y"
+        ],
+        "temporal_coverage": {
+          "start": "2022-01-01T00:00:00Z",
+          "end": "2023-12-31T00:00:00Z"
+        },
+      "versions_url": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions/2023-05/versions",
+      "versions": [
+        {
+          "@id": "https://staging.idpd.uk/datasets/additions-to-the-housing-stock/editions/2023-05/versions/1",
+          "issued": "2023-05-05T08:59:11Z"
+        }
+      ],
+      "next_release": "2024-04-19T09:30:00Z",
+      "table_schema": {
+        "columns": [
+          {
+              "name": "period",
+              "datatype": "string",
+              "titles": "gregorian-interval",
+              "description": "The time period associated with the observation."
+            },
+            {
+              "name": "areacd",
+              "datatype": "string",
+              "titles": "Statistical Geography",
+              "description": "The geography code associated with the observation."
+            },
+            {
+              "name": "areanm",
+              "datatype": "string",
+              "titles": "Area Name",
+              "description": "The name assiciated with The geography code "
+            },
+            {
+              "name": "geography",
+              "datatype": "string",
+              "titles": "Geography Type",
+              "description": "The categorisation of the geography code"
+            },
+            {
+              "name": "variable_name",
```

Comment:
Please remove the column mappings for 
variable_name
indicator
mission
category
polarity
notes

Reasons:
1. They are set to `false` on https://github.com/GSS-Cogs/family-ess/blob/main/additions-to-the-housing-stock/additions-to-the-housing-stock.json
2. There is a maximum number of columns that can be supported by the idpd-poc platform